lib/touch-enroll.js

```diff
@@ -0,0 +1,95 @@
+import { exec as nodeExec } from 'child_process';
+import B from 'bluebird';
+
+// Had problems with teen_process, resorted to this
+const exec = B.promisify(nodeExec);
+
+let touchEnrollMenuKeys = ['Toggle Enrolled State', 'Touch ID Enrolled'];
+let touchEnrollBackups;
```

Comment:
I think this is a special case where we need global state. The `touchEnrollBackups` should only be set once, regardless of how many touchs `enrollTouchId` is called and then it should be restored when all of the simulators are killed.